src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/ApplicationGatewayTests.ps1

```diff
@@ -203,10 +203,10 @@ function Test-ApplicationGatewayCRUD
 		Assert-AreEqual 1 $getgw.Probes[1].Match.StatusCodes.Count
 
 		# Get Application Gateway backend health with expanded resource
-		$job = Get-AzureRmApplicationGatewayBackendHealth -Name $appgwName -ResourceGroupName $rgname -ExpandResource "backendhealth/applicationgatewayresource" -AsJob
-		$job | Wait-Job
-		$backendHealth = $job | Receive-Job
-		Assert-NotNull $backendHealth.BackendAddressPools[0].BackendAddressPool.Name
+		# $job = Get-AzureRmApplicationGatewayBackendHealth -Name $appgwName -ResourceGroupName $rgname -ExpandResource "backendhealth/applicationgatewayresource" -AsJob
```

Comment:
this is just commenting out test code... is there a plan to fix this or is this redundant now?

If its redundant then just remove it.

Reply:
It will be fixed as soon as Ignite work cleans up.